src/cluster/websocket/WebSocket.cc

```diff
@@ -430,6 +423,26 @@ void WebSocketEventDispatcher::HandleSubscriptions(WebSocketClientEntry& entry,
     }
 }
 
+void WebSocketEventDispatcher::HandleSubscriptionsActive(WebSocketClientEntry& entry) {
```

Comment:
Nit: suggest to pass a `const WebSocketClientEntry&` since this does not mutate the value.